Sources/Rendering/OpenGL/VolumeMapper/test/testVolumeMapperBounds.js

```diff
@@ -0,0 +1,91 @@
+import test from 'tape-catch';
+import testUtils from 'vtk.js/Sources/Testing/testUtils';
+
+import 'vtk.js/Sources/Rendering/Misc/RenderingAPIs';
+import vtkHttpDataSetReader from 'vtk.js/Sources/IO/Core/HttpDataSetReader';
+
+import vtkRenderer from 'vtk.js/Sources/Rendering/Core/Renderer';
+import vtkVolume from 'vtk.js/Sources/Rendering/Core/Volume';
+import vtkVolumeMapper from 'vtk.js/Sources/Rendering/Core/VolumeMapper';
+import vtkColorTransferFunction from 'vtk.js/Sources/Rendering/Core/ColorTransferFunction';
+import vtkRenderWindow from 'vtk.js/Sources/Rendering/Core/RenderWindow';
+import vtkRenderWindowInteractor from 'vtk.js/Sources/Rendering/Core/RenderWindowInteractor';
+
+import baseline1 from './testVolumeMapperBounds.png';
+
+test.onlyIfWebGL.only('Test Volume Mapper Bounds', (t) => {
```

Comment:
@sedghi please remove the `.only`.

Reply:
Scratch that, since its the last commit, I'll take care of it.

Reply:
ooops, sorry about that, thanks for taking care of it